.github/workflows/push.yml

```diff
@@ -5,15 +5,16 @@ on:
     branches: [ main ]
 
 jobs:
-  push:
+  publish:
     runs-on: macos-11
+    if: github.repository == 'apollographql/apollo-kotlin'
```

Comment:
See https://github.com/square/kotlinpoet/pull/1119.

Reply:
This could also be added to the `publish` in `tag.yml` too? (This one is for the SNAPSHOTS, whereas the one in `tag` publishes releases).

Reply:
8fa5fe58f8a99f444b79ae14433e1220e3c17f12

Reply:
(just added a small commit to update the job name referenced in a doc)